java/sqlcommenter-java/build.gradle

```diff
@@ -0,0 +1,192 @@
+    googleJavaFormat {
+        toolVersion '1.6'
+    }
+
+    afterEvaluate {  // Allow subproject to add more source sets.
+        tasks.googleJavaFormat {
+            source = sourceSets*.allJava
+            include '**/*.java'
+        }
+
+        tasks.verifyGoogleJavaFormat {
+            source = sourceSets*.allJava
+            include '**/*.java'
+        }
+    }
+}
+
+compileJava {
+    options.compilerArgs += ["-Xlint:none"]
+    options.encoding = "UTF-8"
+}
+
+signing {
+    required false
+    sign configurations.archives
+}
+
+javadoc.options {
+    encoding = 'UTF-8'
+    links 'https://docs.oracle.com/javase/8/docs/api/'
+}
+
+task javadocJar(type: Jar) {
+    classifier = 'javadoc'
+    from javadoc
+}
+
+task sourcesJar(type: Jar) {
+    classifier = 'sources'
+    from sourceSets.main.allSource
+}
+
+artifacts {
+    archives javadocJar, sourcesJar
+}
+
+uploadArchives {
+        repositories {
+            mavenDeployer {
+                beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
+
+                def configureAuth = {
+                    if (rootProject.hasProperty('ossrhUsername') && rootProject.hasProperty('ossrhPassword')) {
+                        authentication(userName:rootProject.ossrhUsername, password: rootProject.ossrhPassword)
+                    }
+                }
+
+                repository(url: "https://oss.sonatype.org/service/local/staging/deploy/maven2/", configureAuth)
+
+                snapshotRepository(url: "https://oss.sonatype.org/content/repositories/snapshots/", configureAuth)
+
+                pom.project {
+                    name "sqlcommenter-hibernate"
+                    packaging 'jar'
+                    description project.description
+                    url 'https://github.com/google/sqlcommenter'
+
+                    scm {
+                        connection 'scm:svn:https://github.com/google/sqlcommenter'
+                        developerConnection 'scm:git:git@github.com/google/sqlcommenter'
+                        url 'https://github.com/google/sqlcommenter'
+                    }
+
+                    licenses {
+                        license {
+                            name 'The Apache License, Version 2.0'
+                            url 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+                        }
+                    }
+
+                    developers {
+                        developer {
+                            id 'com.google.cloud'
+                            name 'Google Developers'
+                            email 'sqlcommenter@googlegroups.com'
+                            url 'cloud.google.com'
+                            // https://issues.gradle.org/browse/GRADLE-2719
+                            organization = 'SQLCommenter Authors'
+                            organizationUrl 'https://google.com'
+                        }
+                    }
```

Comment:
Can changing these be done in next set of commits ?

Reply:
Few similar developer information is there in other languages: python, ruby and nodejs

Reply:
I don't see any problems in changing everything after the initial commit in the follow up clean up PRs

Reply:
Looking at the log-collection commit history stuff like this is addressed in the first commit itself https://github.com/open-telemetry/opentelemetry-log-collection/pull/3.

Reply:
It will be nice, but less important than Copyright. I wouldn't push on all renaming happening in initial commit too hard. If CI can be set up on existing names and renames be made in small targeted PRs, renamings will go smoother and it will be easier to review the new names.

This is my opinion, if you feel strongly, please comment.

Reply:
Yeah that was my thought process, to move in the repository with minimum changes and then create smaller PRs separated by each language working with the CI properly.

Reply:
I don't have any strong opinions, whatever makes it easier for us is fine for me. Let's get this in and divide the further work to transition.

Reply:
@sjs994 I'm not suggesting any renames, I'm suggesting the files should be appropriately copyrighted on import.   Theoretically this can be done with automated tooling, and should have been done in the source repository.

If @SergeyKanzhelev is fine adding copyright after the first commit, that's great.  I'd prefer if the initial import included relevant copyrights on code, even in that code can be renamed/shuffled going forward.

Reply:
Thanks @jsuereth for catching that. 
I missed those files as they had no copyright in the google repository.
Found 9 files which had no copyrights and updated them in the latest commit.